tests/ihontaryk/homework_8/test_person.py

```diff
@@ -0,0 +1,70 @@
+import pytest
+from ihontaryk.homework_8.person import Person
+
+
+@pytest.mark.parametrize('arguments, expected_result',
+                         [(('David', 'Brown', '10.10.1991', 'USA', 'Chicago', '011-444-3333',
+                            'Project Manager', '4.25', '2000.0', 'Male'),
+                           'Full name: David Brown, Age: 33, Gender: Male'
+                           )])
+def test_show_personal_info(arguments, expected_result):
+    """
+    verify show_personal_info function
+    """
+    first_name, last_name, dob, country, city, phone, job_title, experience, salary, *gender = arguments
```

Comment:
why do you treat here to gender as optional if you have only 1 paramterise where gender is present
and parametrise is not needed here